clients/drcachesim/CMakeLists.txt

```diff
@@ -1202,7 +1202,7 @@ if (BUILD_TESTS)
     add_executable(tool.drcacheoff.skip_unit_tests tests/skip_unit_tests.cpp)
```

Comment:
> i#7113 instr decode cache: move module read into raw2trace_shared

Optional suggestion to remove "instr" from all the "instr_decode_cache", "instr decode cache" names and references: in the context of DR or trace analyzers I don't think there's anything else we'd decode so "instr" does not add much and just makes everything longer and possibly harder to understand (I found it harder to then distinguish `instr_decode_cache_t<test_decode_info_t>` and `instr_decode_cache_t<instr_decode_info_t>`: if those were `decode_cache_t<test_decode_info_t>` and `decode_cache_t<decode_info_t>` IMHO that is easier to see the `test_` jump out and easier to read without so many words in there).  IMHO "decode cache" and `decode_cache` are clear and succinct and nicer.

Reply:
Okay, will make that change to #7114 and the PR title.